blocks/editable/index.js

```diff
@@ -907,3 +908,9 @@ Editable.defaultProps = {
 	formattingControls: DEFAULT_FORMATS,
 	formatters: [],
 };
+
+export default withContext( 'editor' )(
+	( settings ) => ( {
+		unfilteredHTML: settings && settings.unfilteredHTML,
```

Comment:
@iseulde Do you think it be better to use the `withAPIData` component and check `user.data.capabilities.unfiltered_html`? [example here](https://github.com/WordPress/gutenberg/blob/e694f6a99b7ce01bb3586798e1af6c662a7a4895/editor/components/post-publish-button/label.js#L63-L69)

Reply:
Good point, didn't think of that. Will try tomorrow.